src/charm.py

```diff
@@ -309,20 +318,10 @@ def join_node_to_cluster(self, event: ops.framework.EventBase) -> None:
 
     def remove_node_from_cluster(self, event: ops.framework.EventBase) -> None:
         """Remove node from microceph cluster."""
-        cmd = ["sudo", "microceph", "cluster", "remove", event.unit.name]
-        try:
-            logger.debug(f'Running command {" ".join(cmd)}')
-            process = subprocess.run(cmd, capture_output=True, text=True, check=True, timeout=180)
-            logger.debug(f"Command finished. stdout={process.stdout}, " f"stderr={process.stderr}")
-        except (subprocess.CalledProcessError, subprocess.TimeoutExpired) as e:
-            logger.warning(e.stderr)
-            # Bugs to be fixed in microceph
-            # https://github.com/canonical/microceph/issues/125
-            # https://github.com/canonical/microceph/issues/49
-            error_remove_node = 'Delete "internal_cluster_members": FOREIGN KEY constraint failed'
-            error_no_node_exists = "No remote exists with the given name"
-            if error_remove_node not in e.stderr and error_no_node_exists not in e.stderr:
-                raise e
+        # NOTE: Since microcluster ops are performed using hostnames, the actual cluster
```

Comment:
Why not completely remove this function?

Reply:
ack, that was an oversight.